CMakeLists.txt

```diff
@@ -871,6 +871,10 @@ if(EXECUTORCH_BUILD_WASM)
   add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/extension/wasm)
 endif()
 
+if(EXECUTORCH_BUILD_TOKENIZERS_WASM)
```

Comment:
I wonder if we have too many cmake options lmao.

Reply:
All of these should belong to tokenizers repo https://github.com/meta-pytorch/tokenizers